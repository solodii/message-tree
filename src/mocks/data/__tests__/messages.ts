import {
  messages,
  messagesById,
  messagesBySubject,
  messagesWithDraft,
  messagesWithoutDraft,
  messageTrees
} from '../messages';

test('messages', () => {
  expect(messages).toMatchSnapshot();
});

test('messagesById', () => {
  expect(messagesById).toMatchSnapshot();
});

test('messagesBySubject', () => {
  expect(messagesBySubject).toMatchSnapshot();
});

test('messagesWithDraft', () => {
  expect(messagesWithDraft).toMatchSnapshot();
});

test('messagesWithoutDraft', () => {
  expect(messagesWithoutDraft).toMatchSnapshot();
});

test('messageTrees', () => {
  expect(messageTrees).toMatchSnapshot();
});
