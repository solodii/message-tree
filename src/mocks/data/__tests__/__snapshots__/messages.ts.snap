// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`messageTrees 1`] = `
Array [
  Object {
    "children": Array [
      Object {
        "children": Array [],
        "hasDraft": false,
        "message": Object {
          "date": 2016-11-10T08:03:00.000Z,
          "id": "913d97b3-6e04-4c1e-a11c-222221abf00e",
          "subjectId": "0a6a7bfd-a212-4ae1-8125-6c7486213a94",
          "text": "third",
        },
      },
    ],
    "hasDraft": true,
    "message": Object {
      "date": 2016-11-10T08:01:00.000Z,
      "id": "0a6a7bfd-a212-4ae1-8125-6c7486213a94",
      "subjectId": "",
      "text": "first",
    },
  },
  Object {
    "children": Array [],
    "hasDraft": false,
    "message": Object {
      "date": 2016-11-10T08:02:00.000Z,
      "id": "6c6f1976-ff5b-420b-bc9d-f7484f0ee53f",
      "subjectId": "",
      "text": "second",
    },
  },
]
`;

exports[`messages 1`] = `
Array [
  Object {
    "date": 2016-11-10T08:01:00.000Z,
    "id": "0a6a7bfd-a212-4ae1-8125-6c7486213a94",
    "subjectId": "",
    "text": "first",
  },
  Object {
    "date": 2016-11-10T08:02:00.000Z,
    "id": "6c6f1976-ff5b-420b-bc9d-f7484f0ee53f",
    "subjectId": "",
    "text": "second",
  },
  Object {
    "date": 2016-11-10T08:03:00.000Z,
    "id": "913d97b3-6e04-4c1e-a11c-222221abf00e",
    "subjectId": "0a6a7bfd-a212-4ae1-8125-6c7486213a94",
    "text": "third",
  },
]
`;

exports[`messagesById 1`] = `
Object {
  "0a6a7bfd-a212-4ae1-8125-6c7486213a94": Object {
    "date": 2016-11-10T08:01:00.000Z,
    "id": "0a6a7bfd-a212-4ae1-8125-6c7486213a94",
    "subjectId": "",
    "text": "first",
  },
  "6c6f1976-ff5b-420b-bc9d-f7484f0ee53f": Object {
    "date": 2016-11-10T08:02:00.000Z,
    "id": "6c6f1976-ff5b-420b-bc9d-f7484f0ee53f",
    "subjectId": "",
    "text": "second",
  },
  "913d97b3-6e04-4c1e-a11c-222221abf00e": Object {
    "date": 2016-11-10T08:03:00.000Z,
    "id": "913d97b3-6e04-4c1e-a11c-222221abf00e",
    "subjectId": "0a6a7bfd-a212-4ae1-8125-6c7486213a94",
    "text": "third",
  },
}
`;

exports[`messagesBySubject 1`] = `
Object {
  "": Array [
    Object {
      "date": 2016-11-10T08:01:00.000Z,
      "id": "0a6a7bfd-a212-4ae1-8125-6c7486213a94",
      "subjectId": "",
      "text": "first",
    },
    Object {
      "date": 2016-11-10T08:02:00.000Z,
      "id": "6c6f1976-ff5b-420b-bc9d-f7484f0ee53f",
      "subjectId": "",
      "text": "second",
    },
  ],
  "0a6a7bfd-a212-4ae1-8125-6c7486213a94": Array [
    Object {
      "date": 2016-11-10T08:03:00.000Z,
      "id": "913d97b3-6e04-4c1e-a11c-222221abf00e",
      "subjectId": "0a6a7bfd-a212-4ae1-8125-6c7486213a94",
      "text": "third",
    },
  ],
}
`;

exports[`messagesWithDraft 1`] = `
Array [
  Object {
    "date": 2016-11-10T08:01:00.000Z,
    "id": "0a6a7bfd-a212-4ae1-8125-6c7486213a94",
    "subjectId": "",
    "text": "first",
  },
]
`;

exports[`messagesWithoutDraft 1`] = `
Array [
  Object {
    "date": 2016-11-10T08:02:00.000Z,
    "id": "6c6f1976-ff5b-420b-bc9d-f7484f0ee53f",
    "subjectId": "",
    "text": "second",
  },
  Object {
    "date": 2016-11-10T08:03:00.000Z,
    "id": "913d97b3-6e04-4c1e-a11c-222221abf00e",
    "subjectId": "0a6a7bfd-a212-4ae1-8125-6c7486213a94",
    "text": "third",
  },
]
`;
